File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/EmailTagObject.cs
using System;
using System.Collections.Generic;

namespace Library.Reader.Objects
{
    public class EmailTagObject
    {
        public EmailTagObject(string company, string mail_to, string mail_cc, string mail_bcc, bool send_flag, List<ReaderTagObject> reader_object_list)
        {
            Company = company;
            MailTo = mail_to;
            MailCc = mail_cc;
            MailBcc = mail_bcc;
            SendFlag = send_flag;

            for (int i = 0; i <= reader_object_list.Count - 1; i++)
            {
                Body += string.Format("<h3>{3}. Location : {1} ({2}) </h3>{0}", Environment.NewLine, reader_object_list[i].Location, reader_object_list[i].Host, i + 1);
                Body += string.Format("<p>Antenna ID : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].AntennaID);
                Body += string.Format("<p>Tag ID : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].TagID);
                Body += string.Format("<p>Time Occured : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].TimeOccured.ToString());
                Body += string.Format("{0}{0}", "<br/>");
            }
        }

        public string Company { get; set; }
        public string MailTo { get; set; }
        public string MailCc { get; set; }
        public string MailBcc { get; set; }
        public bool SendFlag { get; set; }
        public string Body { get; set; }
    }
}

File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/EmailTagObject.vb
Namespace Objects
    Public Class EmailTagObject
        Public Sub New(ByVal company As String, ByVal mail_to As String, ByVal mail_cc As String, ByVal mail_bcc As String, ByVal send_flag As Boolean, ByVal reader_object_list As List(Of ReaderTagObject))

            Me.Company = company
            Me.MailTo = mail_to
            Me.MailCc = mail_cc
            Me.MailBcc = mail_bcc
            Me.SendFlag = send_flag

            For i As Integer = 0 To reader_object_list.Count - 1

                Me.Body &= String.Format("<h3>{3}. Location : {1} ({2}) </h3>{0}", Environment.NewLine, reader_object_list(i).Location, reader_object_list(i).Host, i + 1)
                Me.Body &= String.Format("<p>Antenna ID : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).AntennaID)
                Me.Body &= String.Format("<p>Tag ID : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).TagID)
                Me.Body &= String.Format("<p>Time Occured : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).TimeOccured.ToString)
                'Me.Body &= String.Format("<p>Message : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).Message)

                Me.Body &= String.Format("{0}{0}", "<br/>")

            Next

        End Sub

        Public Property Company As String

        Public Property MailTo As String

        Public Property MailCc As String

        Public Property MailBcc As String

        Public Property SendFlag As Boolean

        Public Property Body As String
    End Class

End Namespace

